modules/460-log-shipper/hooks/internal/composer/composer.go

```diff
@@ -30,12 +32,14 @@ import (
 type Composer struct {
 	Source []v1alpha1.ClusterLoggingConfig
 	Dest   []v1alpha1.ClusterLogDestination
+	Token  string
```

Comment:
The composer should not know about the token. It is not a composer property.

Reply:
The better way is to add it to the loki snapshot while reading CLD.

Reply:
done